elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -1,79 +1,140 @@
 @import "../../pfe-sass/pfe-sass";
 
+$USE-FALLBACK: true;
 $LOCAL: cta;
+
 $pfe-cta__arrow-size: 0.313em;
 $pfe-cta--BackgroundColor--focus: rgba(40,151,240, .2);
 $pfe-cta--Color--fallback: #003366;
 
 :host {
-  // regular variables
+  --pfe-cta--Padding:      .6rem 0;
   --pfe-cta--BorderRadius: 0;
-  --pfe-cta__arrow--Display:  inline;
-  --pfe-cta__arrow--Padding:  0 .125rem 0 .375rem;
+
+  --pfe-cta__arrow--Display:     inline;
+  --pfe-cta__arrow--Padding:     0 .125rem 0 .375rem;
+  --pfe-cta__arrow--MarginLeft:  calc(#{pfe-var(content-spacer)} * 0.25);
+  
   --pfe-cta--BackgroundColor:         transparent;
+  --pfe-cta--BackgroundColor--hover:  transparent;
   --pfe-cta--BackgroundColor--focus:  transparent;
-  --pfe-cta--SecondaryBackgroundColor--focus: rgba(21,21,21, .05);
-  --pfe-cta--BorderColor:             transparent;
-  --pfe-cta__inner--BorderColor:      transparent;
-  --pfe-cta--Color:   #{pfe-color(ui-link)};
-  --pfe-cta--Padding: .6rem 0;
 
-  // hover variables
-  --pfe-cta--BackgroundColor--hover:  transparent;
-  --pfe-cta--BorderColor--hover: transparent;
-  --pfe-cta--Color--hover:  #{pfe-color(ui-link--hover)};
+  --pfe-cta--BorderColor:           transparent;
+  --pfe-cta--BorderColor--hover:    transparent;
+  --pfe-cta--BorderColor--focus:    transparent;
 
-  // focus variables
-  --pfe-cta--BorderColor--focus: transparent;
+  --pfe-cta__inner--BorderColor:        transparent;
```

Comment:
needed:
```
--pfe-cta__inner--BorderColor--focus:  
--pfe-cta__inner--BorderColor--hover:  
```

Reply:
@starryeyez024 There is no inner border hover state so no need to have a variable for that and the focus state is on line 27 already.

Reply:
ah thx